TestON/core/teston.py

```diff
@@ -158,7 +159,7 @@ def componentInit(self,component):
         driver_options ['type'] = driverName
         
         classPath = self.getDriverPath(driverName.lower())
-        driverModule = importlib.import_module(classPath)
+        driverModule = __import__(classPath, globals(), locals(), [driverName.lower()], -1)
```

Comment:
Hi Jon,
  Could you provide some details about this change?
Also the need for additional args "globals(), locals(), [driverName.lower()], -1".
I notice that classPath already calls self.getDriverPath(driverName.lower()).